crates/compiler/build/src/program.rs

```diff
@@ -277,7 +277,7 @@ fn gen_from_mono_module_llvm(
         interns: loaded.interns,
         module,
         target_info,
-        mode: LlvmBackendMode::Binary,
+        mode: LlvmBackendMode::BinaryDev,
```

Comment:
hardcoded for now; this should be dynamic and I'd like to do it based on the optlevel (where we have a Development option) but that means `Development` no longer means "dev backend"